MEDO v0.5.owl

```diff
@@ -804,6 +975,536 @@ LCME Glossary of Terms for LCME Accreditation Standards and Elements 2015-2016,
+        <obo:IAO_0000119>MedBiquitous - AM17: The process of evaluating one’s own deficiencies, achievements, behavior or professional performance and competencies (Institute for International Medical Education, 2002); Assessment completed by the learner to reflect and critically assess his/her own performance against a set of established criteria (Gordon, 1991) (NOTE: Does not refer to NBME Self-Assessment)
+
+Synonymous with/Includes: Observation of self on video
+
+Does not include: NBME Self-Assessment (National Board of Medical Examiners (NBME) Self-Assessment Services, n.d.)</obo:IAO_0000119>
+        <rdfs:label>formal formative self-assessment</rdfs:label>
+        <skos1:related>MedBiquitous - AM17: Self-Assessment</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#formal_formative_teacher_observation_and_feedback -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#formal_formative_teacher_observation_and_feedback">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo//MEDO_0000103"/>
+        <obo:IAO_0000115>A type of formal formative assessment where teacher or observer(s) observes, assesses, and provides written or oral subjective feedback of a learner&apos;s work or performance.</obo:IAO_0000115>
+        <obo:IAO_0000119>MedBiquitous - AM10 - Narrative Assessment: An instructor&apos;s or observer&apos;s written subjective assessment of a learner&apos;s work or performance (Mennin, McConnell, &amp; Anderson, 1997); May Include: Comments within larger assessment; Observation of learner or small group by instructor
+
+Does not include: Clinical Documentation Review; Clinical Performance Rating/Checklist; Peer Assessment; Self-Assessment</obo:IAO_0000119>
+        <rdfs:label>formal formative teacher observation and feedback</rdfs:label>
+        <skos1:related>MedBiquitous - AM10: Narrative Assessment</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#formal_summative_oral_patient_presentation -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#formal_summative_oral_patient_presentation">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo//MEDO_0000105"/>
+        <obo:IAO_0000115>A type of formal summative assessment where a student presents clinical case (patient) findings, history and physical, differential diagnosis, treatment plan, etc., to an instructor or small group for the purpose of demonstrating skills in clinical reasoning and problem-solving and discussing the experience, process, and performance.</obo:IAO_0000115>
```

Comment:
I prefer "A type of formal summative assessment in which a student presents clinical case (patient) findings, history and physical, differential diagnosis, treatment plan, etc., to an instructor or small group for the purpose of demonstrating skills in clinical reasoning and problem-solving with a subsequent discussion of the experience, process, and performance.